src/status_printer.cc

```diff
@@ -49,8 +49,14 @@ StatusPrinter::StatusPrinter(const BuildConfig& config)
     printer_.set_smart_terminal(false);
 
   progress_status_format_ = getenv("NINJA_STATUS");
-  if (!progress_status_format_)
+
+  string error_output;
```

Comment:
nit: use `std::string` as we are trying to remove `using namespace std` from the sources gradually.